### ISSUE TITLE
Table-Valued parameters for SQL server

### ISSUE THREAD

**Issue by dmitrijsc** [JuliaDB/JDBC.jl#25]
Hi guys

Any plans to implement TVP for SQL server to support bulk load?

https://docs.microsoft.com/en-us/sql/connect/jdbc/using-table-valued-parameters#table-valued-parameter-api-for-the-jdbc-driver

**Comment by aviks** [comment]
I hadn't seen this functionality before, doesn't seem like a standard JDBC facility, seems specific to SQL server. Which is not to say it can't be supported, just that it will take some investigation, and I can't promise when I'll be able to get to that. 

For the moment, it might be possible to get to that via direct `JavaCall` invocations.

**Comment by dmitrijsc** [comment]
You are correct. It is specific to SQL server. Hope you will find a time to look into it :)
